CHANGES.rst

```diff
@@ -1,6 +1,16 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-19)
+--------------------------
+
+- Adds new method which returns full workflow name.
+- Adds more granular DB configuration.
+- Adds ``git_repo``, ``git_provider``  to the workflow model.
```

Comment:
Note that to easy identify model and columns, we used a policy to write free phrase followed by `(Model.Field)``, see the release notes of v0.5.0.  I'll amend the text accordingly...